app/bundles/LeadBundle/Model/LeadModel.php

```diff
@@ -2234,4 +2234,16 @@ public function getLeadEmailStats(Lead $lead): array
 
         return $statRepository->getStatsSummaryForContacts([$lead->getId()])[$lead->getId()];
     }
+
+    public function removeTagFromLead(int $leadId, int $tagId): void
+    {
+        /** @var \Mautic\LeadBundle\Entity\Lead $lead */
```

Comment:
```suggestion
```
This is incorrect as it can return null. The method `getEntity()` already has the right return type. Let's remove this line.